validation/src/main/scala/hmda/validation/engine/lar/syntactical/LarSyntacticalEngine.scala

```diff
@@ -1,19 +1,29 @@
 package hmda.validation.engine.lar.syntactical
 
 import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.model.institution.Institution
 import hmda.validation.api.ValidationApi
+import hmda.validation.context.ValidationContext
 import hmda.validation.engine.lar.LarCommonEngine
+import hmda.validation.rules.ts.syntactical.S025
 import hmda.validation.rules.lar.syntactical._
 
 trait LarSyntacticalEngine extends LarCommonEngine with ValidationApi {
 
-  def checkSyntactical(lar: LoanApplicationRegister): LarValidation = {
+  private def s025(lar: LoanApplicationRegister, ctx: ValidationContext): LarValidation = {
+    convertResult(lar, S025(lar, ctx), "S025")
+  }
+
+  def checkSyntactical(lar: LoanApplicationRegister, ctx: ValidationContext): LarValidation = {
     val checks = List(
       S010,
       S020,
       S205
     ).map(check(_, lar))
 
+    // Exclude edits requiring institution data
+    if (ctx.institution.isDefined) checks :+ s025(lar, ctx)
```

Comment:
This line does not appear to have any effect. Same for the analogous one in `TsSyntacticalEngine`.  I now lean even more strongly toward trying another way (which I've started, but don't have a PR yet).

We'll also need tests for the intended functionality here, and there.

Side note: TIL a one-line `if` means codecov won't check whether it happens.  Too bad, but I suppose it has to be that way (because that's what line coverage means), and it's good to know!